apps/www/registry/default/ui/spinner.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react"
+
+import { cn } from "@/lib/utils"
+
+const Spinner = React.forwardRef<
+  HTMLDivElement,
+  React.ComponentPropsWithoutRef<"div">
+>(({ className, ...props }, ref) => {
+  const computeDelay = (i: number) => `${-1.2 + i * 0.1}s`
+  const computeRotation = (i: number) => `${i * 30}deg`
+  return (
+    <div
+      className={cn("size-5", className)}
+      role="status"
+      aria-label="Loading"
+      ref={ref}
+      {...props}
+    >
+      <div className="relative left-1/2 top-1/2 size-full">
+        {[...Array(12)].map((_, i) => (
```

Comment:
@flixlix 

```ts
{[...Array<number>(12)].map((_, i) => (
```

If you explicitly specify the number type as shown above, it is better because TypeScript errors will not occur depending on the environment.

Incidentally, on my environment, if I did not specify the number type, I got an error `ESLint: Unsafe spread of an `any` value in an array.(@typescript-eslint/no-unsafe-assignment)`.

Reply:
fixed :)